MobileBuy/sample/src/main/java/com/shopify/sample/view/product/ProductDetailActivity.java

```diff
@@ -106,69 +110,75 @@
     initViewModels(productId);
 
     imageGalleryView.renderImages(Arrays.asList(productImageUrl));
-    swipeRefreshLayoutView.setOnRefreshListener(() -> productViewModel.refetch());
+    swipeRefreshLayoutView.setOnRefreshListener(() -> productDetailViewModel.fetchData());
     productDescriptionView.renderProduct(productTitle, productPrice);
-    productDescriptionView.setOnAddToCartClickListener(() -> productViewModel.addToCart());
+//    productDescriptionView.setOnAddToCartClickListener(() -> productDetailViewModel.addToCart());
```

Comment:
I am working on adding share CartViewModel so that cart data will be consistent throughout the app.